src/pe-simple-header/widget/pe-simple-header.u-link-script.js

```diff
@@ -0,0 +1,44 @@
+function ($scope,$el,$attrs,$ctrl) {
```

Comment:
- [ ] remove the following:

1. $scope
2. $attr
3. $ctrl

They are not used.

Reply:
This is standard and best practice for the link function when you are actually 'using' the link function.  These 4 objects are passed into the link function and should be referenced in the link definition whether used or not used in the function itself.

Reply:
Question: does removing the above dependencies effect functionality of the widget or performance?

Reply:
Answer: Rhetorical - Does keeping the dependencies effect functionality or the widget or performace? --- Both answers are "NO" because regardless of the link function's definition the objects are all instantiated as part of the controller - $scope == $scope, $el == widget, $attr == widget's attributes, and $ctrl is the controller.  The simplicity of them being referenced in link functions definition doesn't change functionality or performance.  Unlike declaring dependencies is the controller function's definition which actual does an instantiation of the dependency for use.

Reply:
thank you for the clarification.